### DISCUSSION TITLE
I can't access a nginx server ran with podman.

### DISCUSSION THREAD

**Discussion by Luxvao** [podman-desktop/podman-desktop#719] [Q&A] [answered]
I'm running podman desktop and I tried to run an nginx server on the port 8080, but when I try to access it with localhost:8080 I get that the connection was reset. I don't know if this is a bug so I posted it here.

**Comment by benoitf** [comment, marked as answer]
Hello could you share your run command ?

**Reply by Luxvao** [reply]
podman run -p 8080:8080 --name nginx nginx

**Reply by benoitf** [reply]
nginx is listening on port 80, not 8080

https://hub.docker.com/_/nginx
![image](https://user-images.githubusercontent.com/436777/198529958-3ed9edbb-beff-42f5-8015-d554b79e9ba8.png)


Use
```bash
podman run -p 8080:80 --name nginx nginx
```

**Reply by Luxvao** [reply]
Oh okay thanks
